mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -393,13 +393,20 @@
         
         "MOM_Settings_SJ_Enable_Trail_Particle" "Enable stickybomb trail particle"
         "MOM_Settings_SJ_Enable_Explosion_Particle" "Enable stickybomb explosion particle"
+        "MOM_Settings_SJ_Enable_Stickybomb_Decals" "Enable stickybomb explosion decals"
+        "MOM_Settings_SJ_Enable_Stickybomb_Decals_TT" "When enabled, stickybombs will create decals upon explosion."
         "MOM_Settings_SJ_Enable_Explosion_Sound" "Enable stickybomb explosion sound"
         "MOM_Settings_SJ_Enable_Detonation_Fail_Sound" "Enable stickybomb detonation fail sound"
         "MOM_Settings_SJ_Enable_Detonation_Fail_Sound_TT" "Note: if this is disabled and success sound is enabled, success sound will play if partial detonation happens."
         "MOM_Settings_SJ_Enable_Detonation_Success_Sound" "Enable stickybomb detonation success sound"
         "MOM_Settings_SJ_Enable_Charge_Sound" "Enable sticky launcher charging sound"
         "MOM_Settings_SJ_Enable_ChargeMeter" "Enable charge meter"
+        "MOM_Settings_SJ_ChargeMeter_Units_Label" "Chargemeter Units:"
+        "MOM_Settings_SJ_ChargeMeter_Units_Type_None" "None"
+        "MOM_Settings_SJ_ChargeMeter_Units_Type_1" "Hammer units per second (U/S)"
+        "MOM_Settings_SJ_ChargeMeter_Units_Type_2" "Speed in percent"
         "MOM_Settings_SJ_Enable_Sticky_Counter" "Enable stickybomb counter"
+        "MOM_Settings_SJ_Enable_Sticky_Counter_Autohide" "Enable autohide on stickybomb counter"
```

Comment:
It may be good to have a tooltip that clarifies this; autohide is when the count is 0

Reply:
Good idea, added that :)